contrib/wsgi/moin2.wsgi

```diff
@@ -26,22 +26,23 @@ moin_dir = '/path/to/moin/dir/where/wikiconfig.py/is/located'
 if sys.platform == 'win32':
     site.addsitedir(moin_dir + '-venv-python/Lib/site-packages')
 else:
+# TODO: the venv is created in ../moin-venv-python3.5 by quickinstall...
```

Comment:
so, this is a bug because there is "3.5" missing after "python"?

if so, please do a separate commit or a bug report, it doesn't fit into a "fix the prints" commit.

Reply:
You are right, sorry - moved this issue to https://github.com/moinwiki/moin/issues/929. The comment will be removed from the PR...